src/components/reports/cost.test.ts

```diff
@@ -206,7 +208,7 @@ describe('cost report test suite', () => {
       {
         eventGUID: '', eventStart: new Date(), eventStop: new Date(),
         resourceGUID: '', resourceName: '', resourceType: '', orgGUID: '',
-        spaceGUID: '', spaceName: '', planGUID: '', numberOfNodes: 0, memoryInMB: 0,
+        spaceGUID: '', quotaGUID: '', spaceName: '', planGUID: '', numberOfNodes: 0, memoryInMB: 0,
```

Comment:
Could we pull creating these objects out in to a helper function that mixes-in its input? That should help tidy up these tests a bit, and make them easier to maintain in the long run.

```
function newBillableEvent(overrides){
    return mixin({billable: event}, overrides);
}

const summed = reports.sumRecords([
    newBillableEvent({price: {incVat: 10}}),
    newBillableEvent({price: {incVat: 5.5}}),
]);
```

Reply:
There's nice syntax for this in the language even:

```
const defaultBillableEvent = { eventGuid: '', /*... and the rest */ }

const summed = reports.sumRecords([
  {...defaultBillableEvent, price: {incVat: 10}},
  {...defaultBillableEvent, price: {incVat: 5.5}},
])
```

(or something like that)